warehouse/templates/manage/project/history.html

```diff
@@ -167,9 +172,16 @@ <h2>{% trans %}Security history{% endtrans %}</h2>
     {% elif event.tag == EventTag.Project.APITokenAdded %}
     <strong>{% trans %}API token added{% endtrans %}</strong>
```

Comment:
I think we probably want a different heading here entirely for these, something like "Short-lived API token created"

Reply:
Sounds good. In that case, were you thinking they'd have an entirely separate `EventTag.Project` value?

Reply:
If that's easier, yeah let's do it.

Reply:
Would make it easier for us to filter different types of token events in the future as well.

Reply:
Yep, I think that'll be easiest. Doing now.

Reply:
[0fb4679](https://github.com/pypi/warehouse/pull/11272/commits/0fb4679ca2394e08389e8a284910bdcb33b7f95e)